### DISCUSSION TITLE
Entry level conditions for EntryType

### DISCUSSION THREAD

**Discussion by scoch5** [craftcms/cms#14605] [Ideas]
Have the ability to apply conditions (a bit like you can do for fields) based on the entry level.

**Comment by lennonpuype** [comment]
I was going to open a discussion for this.

Would be really handy since we were used to use NEO fields for what a matrix now can do. It would be very usefull to have these conditions for Entry types inside a matrixfield to for example disable some Entry types for certain sections.

For now I can just create another field and add the entry types needed for that section.

But love to see this happening... 👀

**Comment by andrewhawkes** [comment]
I'm not sure if this should be a different discussion or not - but additionally, it would be nice to be able to do the following with the entry types within matrix fields:

1. Set a limit of how many times an entry type can be used
2. Ability to group entry types within the dropdown list and/or have different lists for each group like Neo

**Reply by brandonkelly** [reply]
Grouped entry types are being discussed over at #14600.

**Comment by russback** [comment]
+1 for this.

Having pages where you can build out all manner of content components is awesome, but sometimes we need to give editors a little bit of guidance on particular pages where only certain components are appropriate, mainly for design consistency.

Rather than having multiple matrix fields like this:

- Unlimited Options (all entry types)
- Landing Page (selected 6 entry types)
- Home Page (selected 4 entry types)

It would be great if we could set which entry types are available to choose based on the section you are in, using a conditional in the field setting so the section's entry type layout.
